include/micm/process/process_set.hpp

```diff
@@ -21,7 +21,9 @@ namespace micm
     std::vector<std::size_t> number_of_products_;
     std::vector<std::size_t> product_ids_;
     std::vector<double> yields_;
-    std::vector<std::size_t> jacobian_flat_ids_;
+    //std::vector<std::size_t> jacobian_flat_ids_;
```

Comment:
Please either make it private and add a getter or just delete this commented out line if it needs to be public

Reply:
Thanks for catching this. I will make it to be private.